docs/insomnia/declarative-config.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Is it worth linking to the O2K docs from this page (until/if we move them onto this docs site eventually)? https://github.com/Kong/insomnia/tree/develop/packages/openapi-2-kong#kong-declarative-config